core/src/main/codegen/templates/Parser.jj

```diff
@@ -7106,6 +7142,7 @@ SqlPostfixOperator PostfixRowOperator() :
 |   < XML: "XML" >
 |   < YEAR: "YEAR" >
 |   < ZONE: "ZONE" >
+|   <TUMBLE: "TUMBLE">
```

Comment:
https://twitter.com/julianhyde/status/1174049701496143872 is all

Reply:
oops. Updated.